ipywidgets/static/widgets/js/widget_selectioncontainer.js

```diff
@@ -160,7 +160,7 @@ define([
 
                 // Trigger the displayed event of the child view.
                 that.displayed.then(function() {
-                    view.trigger('displayed');
+                    view.trigger('displayed', that);
```

Comment:
Widgets and WidgetModels don't have a notion of parent, this is as-designed.  However, DOM nodes have a notion of parent and children yet DOMWidgetViews do not.  By adding that, I'm adding the idea of a parent view to the DOMWidgetView.  I though this made sense, because the alternative would be to make the StyleView have an invisible el which gets attached to the DOM, and then use that to get the parent.  All of this is because the Style applies to the parent.